src/docs/metis.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: Metis
+subtitle: Metis is an EVM-compatible Optimistic Rollup. The protocol team centrally operates the sequencer. Transaction data is recorded off-chain in MEMO distributed data storage system. The protocol provides an optimistic challenge mechanism that allows Validators to force Sequencer to post missing data, however it is not fully implemented yet. Their contracts are based on the optimistic virtual machine (OVM).
+labels:
+    - Optimistic Rollup
+    - MVM
+    - Mainnet
+links:
+    website:
+        url: https://www.metis.io/
+        name: metis.io
+    docs:
+        url: https://docs.metis.io/dev/
+        name: Docs
+    l2beat:
+        url: https://l2beat.com/scaling/projects/metis
+        name: L2Beat
+    github:
+        url: https://github.com/MetisProtocol
+        name: Github
+    twitter:
+        url: https://twitter.com/MetisDAO
+        name: MetisDAO
+    discord:
+        url: https://discord.com/invite/RqfEJZXnxd
+        name: Discord
+---
+
+<Section title="Overview">
+
+    Metis is an EVM-compatible Optimistic Rollup. The protocol team centrally operates the sequencer. Transaction data is recorded off-chain in MEMO distributed data storage system. The protocol provides an optimistic challenge mechanism that allows Validators to force Sequencer to post missing data, however it is not fully implemented yet. Their contracts are based on the optimistic virtual machine (OVM).
+
+    ###### Focus
+
+    - Finalization of the validator challenge mechanism.
+
+    <Labels labels={['Optimistic Rollup', 'MVM', 'Mainnet']} title="Metis" />
```

Comment:
```suggestion
    <Labels labels={['Optimistic Rollup', 'EVM', 'Mainnet']} title="Metis" />
```

Reply:
I would make it EVM since they are basically a modified EVM. The same way any other rollup in the current list is a modified EVM.